open_spiel/game_transforms/repeated_game.h

```diff
@@ -65,6 +65,8 @@ class RepeatedGame : public SimMoveGame {
                const GameParameters& params);
   std::unique_ptr<State> NewInitialState() const override;
   int MaxGameLength() const override { return num_repetitions_; }
+  // There aren't chance nodes in this game.
+  int MaxChanceNodesInHistory() const override { return 0; }
```

Comment:
I think this is wrong

Reply:
Based on https://github.com/deepmind/open_spiel/blob/master/open_spiel/game_transforms/repeated_game.cc#L188

Reply:
I will drop it, as now we change it to return 0 if game is deterministic by default.